examples/hosted-delivery-function-potterdb/delivery-functions/index.ts

```diff
@@ -0,0 +1,42 @@
+import { type EventHandler, EventType } from './types'
+
+const fieldMappingHandler: EventHandler<EventType.GRAPHQL_FIELD_MAPPING> = (event, context) => {
+  const fields = event.fields.map(({ contentTypeId, field }) => {
+    return {
+      contentTypeId,
+      fieldId: field.id,
+      graphQLOutputType: 'Character',
+      graphQLQueryField: 'character',
+      graphQLQueryArguments: { slug: '' },
+    }
+  })
+
+  return {
+    namespace: 'PotterDB',
+    fields,
+  }
+}
+
+const queryHandler: EventHandler<EventType.GRAPHQL_QUERY> = async (event, context) => {
+  const response = await fetch('https://api.potterdb.com/graphql', {
```

Comment:
I would add a comment here about `isIntrospectionQuery` and how you're forwarding both to the api. Introspection query is the first thing that breaks usually 😅

Reply:
Added a comment. Lmk if it addresses what you had in mind.